packages/cozy-dataproxy-lib/package.json

```diff
@@ -68,6 +67,7 @@
     "cozy-intent": ">=2.26.0",
     "cozy-logger": ">=1.10.4",
     "cozy-minilog": ">=3.3.1",
+    "cozy-pouch-link": "50.3.1",
```

Comment:
Declaring it in peerDeps is not enough, you should also declare it in devDeps

Reply:
It is already declared in devDep: https://github.com/cozy/cozy-libs/blob/623a5c6aee86cba03d652950d22870fc9237156d/packages/cozy-dataproxy-lib/package.json#L35C6-L35C21

Reply:
My bad 😅